WordPress/Classes/ViewRelated/System/WPTabBarController+ShowTab.swift

```diff
@@ -1,3 +1,4 @@
+import WordPressKit
```

Comment:
Not sure why these changes appeared here. I will push another commit to fix this (this file should have no changes from `trunk` now).

Reply:
Resolved w/ https://github.com/wordpress-mobile/WordPress-iOS/pull/17429/commits/ecff147783328f0cb1d2a448716ae003bff46b77